packages/matter.js/test/device/MatterNodeStructureTest.ts

```diff
@@ -296,7 +296,7 @@ describe("Endpoint Structures", () => {
             assert.deepEqual(rootPartsListAttribute?.getLocal(), [new EndpointNumber(1)]);
 
             assert.equal(attributePaths.length, 146);
-            assert.equal(commandPaths.length, 38);
+            assert.equal(commandPaths.length, 39);
```

Comment:
And that's the problem. There should be. I increase here. Only happens because we now have "not implemented" commands reported as supported commands. See above

Reply:
My bad.  When I auto-registered the clusters a bunch of numbers changed, I think I got clusters and commands confused.

Reply:
I more assume it comes because of the "implementing optional commands with "no implementation"

Reply:
Yeah that's what caused it